snapcraft/meta.py

```diff
@@ -230,17 +190,13 @@ def _wrap_exe(relexepath, args=None):
     wrappath = exepath + '.wrapper'
     shebang = None
 
-    # TODO talk to original author if the exception to be captured here is
-    # FileNotFoundError, the original code was a general catch all
-    try:
+    with contextlib.suppress(FileNotFoundError):
```

Comment:
Would it be more clear if you simply checked for the presence of the file before removing it?